pyfftw/builders/_utils.py

```diff
@@ -60,18 +60,35 @@
 _valid_efforts = ('FFTW_ESTIMATE', 'FFTW_MEASURE',
         'FFTW_PATIENT', 'FFTW_EXHAUSTIVE')
 
-# Looking up a dtype in here returns the complex complement of the same
-# precision.
+# Looking up a real dtype in here returns the complex complement of the same
+# precision, and vice versa.
 # It is necessary to use .char as the keys due to MSVC mapping long
 # double to double and the way that numpy handles this.
-_rc_dtype_pairs = {numpy.dtype('float32').char: numpy.dtype('complex64'),
+_rc_dtype_pairs = {}
+_default_dtype = None
+
+# Double precision is the default. Prefer casting to higher precision if
```

Comment:
The default default :wink:

Reply:
Yes, I should make that my default default default